Source/WebCore/html/HTMLOptionElement.h

```diff
@@ -64,7 +66,10 @@ class HTMLOptionElement final : public HTMLElement {
 
     String textIndentedToRespectGroupLabel() const;
 
+    // Update 'selectedness'.
     void setSelectedState(bool, AllowStyleInvalidation = AllowStyleInvalidation::Yes);
+    // Update 'dirtiness'.
```

Comment:
This should be done by selecting an appropriate function name, not with a comment. Like name it `setDirtiness` and then the comment isn’t needed.